### PR TITLE
Support fragment navigation

### PR DESCRIPTION
**Pull request by kiootic**

### CODE REVIEW

File: sdk/src/main/java/com/oursky/authgear/latte/fragment/LatteAppLink.kt [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+                    conn.errorStream?.use {
+                        val responseString = String(it.readBytes(), StandardCharsets.UTF_8)
+                        HttpClient.throwErrorIfNeeded(conn, responseString)
+                    }
+                    conn.inputStream.use {
+                        val responseString = String(it.readBytes(), StandardCharsets.UTF_8)
+                        HttpClient.throwErrorIfNeeded(conn, responseString)
+                    }
+                }
+            }
```
Comment (tung2744) [inline sdk/src/main/java/com/oursky/authgear/latte/fragment/LatteAppLink.kt:40]: Seems now we have activity link handler and fragment link handler. Should we extract the common logic from them to ensure they are same in both implementation?

Comment (kiootic) [reply sdk/src/main/java/com/oursky/authgear/latte/fragment/LatteAppLink.kt:40]: I was thinking of deprecating the activity applink implementation and remove it later, or refactor activity applink to use fragment applink.

Comment (louischan-oursky) [reply sdk/src/main/java/com/oursky/authgear/latte/fragment/LatteAppLink.kt:40]: Choose whichever approach that takes the least effort.

Comment (kiootic) [reply sdk/src/main/java/com/oursky/authgear/latte/fragment/LatteAppLink.kt:40]: I'll delete the activity applink and flows in another PR.

**Pull request merged**